docs/sources/datasources/influxdb/_index.md

```diff
@@ -203,9 +203,8 @@ datasources:
     access: proxy
     url: http://localhost:8086
     jsonData:
-      version: SQL
```

Comment:
Oh wait I noticed something. This `version: SQL` part should to SQL example. Could you please make that change too?

Reply:
> Oh wait I noticed something. This `version: SQL` part should to SQL example. Could you please make that change too?

Sorry, I'm not following your comment. Are you saying that you want version: SQL to remain in the example and I should undo this change?

Reply:
ping @itsmylife

Reply:
Below InfluxDB 3.x for SQL Example `jsonData` must have `version: SQL`. The change you made here `InfluxDB 2.x for InfluxQL example` is correct and no need additional changes.

Reply:
```
datasources:
  - name: InfluxDB_v3_InfluxQL
    type: influxdb
    access: proxy
    url: http://localhost:8086
    jsonData:
      version: SQL  <---------- THIS LINE MUST BE ADDED
      dbName: site
      httpMode: POST
    secureJsonData:
      token: '<api-token>'
```

Reply:
> ```
> datasources:
>   - name: InfluxDB_v3_InfluxQL
>     type: influxdb
>     access: proxy
>     url: http://localhost:8086
>     jsonData:
>       version: SQL  <---------- THIS LINE MUST BE ADDED
>       dbName: site
>       httpMode: POST
>     secureJsonData:
>       token: '<api-token>'
> ```

Ok, thanks! gotchya! I've updated the pull request and added version: SQL to the v3 example